dkpro-core-io-bincas-asl/src/main/java/de/tudarmstadt/ukp/dkpro/core/io/bincas/BinaryCasReader.java

```diff
@@ -47,169 +57,234 @@
-     * the type system ends in {@code .ser} and the resources also end in {@code .ser}. If this is
-     * the case, we filter the type system file from the resource files during scanning.
-     */
-    @Override
-    protected Collection<Resource> scan(String aBase, Collection<String> aIncludes,
-            Collection<String> aExcludes)
-        throws IOException
-    {
-        Collection<Resource> resources = super.scan(aBase, aIncludes, aExcludes);
-        if (typeSystemLocation != null) {
-            org.springframework.core.io.Resource r = getTypeSystemResource();
-            resources.remove(new Resource(null, null, r.getURI(), null, null, r));
-        }
-        return resources;
-    }
-    
-    protected org.springframework.core.io.Resource getTypeSystemResource() throws MalformedURLException
-    {
-        org.springframework.core.io.Resource r;
-        // Is absolute?
-        if (typeSystemLocation.indexOf(':') != -1 || typeSystemLocation.startsWith("/")
-                || typeSystemLocation.startsWith(File.separator)) {
-            // If the type system location is absolute, resolve it absolute
-            r = getResolver().getResource(locationToUrl(typeSystemLocation));
-        }
-        else {
-            // If the type system is not absolute, resolve it relative to the base location
-            r = getResolver().getResource(getBase() + typeSystemLocation);
-        }
-        return r;
-    }
-    
-    private TypeSystemImpl readTypeSystem() throws IOException {
-        if (typeSystemLocation == null) {
-            return null;
-        }
-        
-        if (typeSystem == null) {
-            CASMgrSerializer casMgr = readCasManager();
-            typeSystem = casMgr.getTypeSystem();
-            typeSystem.commit();
-        }
-        
-        return typeSystem;
-    }
-    
-    
-    private CASMgrSerializer readCasManager() throws IOException
-    {
-        if (typeSystemLocation == null) {
-            return null;
-        }
-        
-        // If we already read the type system, return it - do not read it again.
-        if (casMgrSerializer != null) {
-            return casMgrSerializer;
-        }
-        
-        org.springframework.core.io.Resource r = getTypeSystemResource();
-        getLogger().debug("Reading type system from [" + r.getURI() + "]");
-
-        ObjectInputStream is = null;
-        try {
-            is = new ObjectInputStream(CompressionUtils.getInputStream(typeSystemLocation, 
-                    r.getInputStream()));
-            casMgrSerializer = (CASMgrSerializer) is.readObject();
-        }
-        catch (ClassNotFoundException e) {
-            throw new IOException(e);
-        }
-        finally {
-            closeQuietly(is);
-        }
-        
-        
-        return casMgrSerializer;
-    }
+public class BinaryCasReader extends ResourceCollectionReaderBase {
```

Comment:
It seems that there were a lot of changes to whitespace. Could you please revert all changes that are whitespace-only, such that it becomes possible to see the *actual* differences between the original code and the modified code?